js/demo-app/src/workers/demo-worker.js

```diff
@@ -0,0 +1,24 @@
+import BackgroundClient from "iframe-coordinator/BackgroundClient";
+// TODO Need a better demo that makes more sense?
+
+const client = new BackgroundClient();
```

Comment:
Took a fair amount of webpack finagling to get this to bundle correctly as a worker.  But the end result is cool!  All the messaging infrastructure is hidden behind the BackgroundClient api.

Reply:
Can we make this a non-default export?  It would match our other entry points and let us potentially expand the exports eaiser.